src/Transform/ONNX/Decompose.cpp

```diff
@@ -199,6 +199,60 @@ void populateDecomposingONNXBeforeMhloPatterns(
 }
 #endif
 
+// Special Op fusion for the following pattern:
+//   %1 = Concat(inputs, axis)
+//   %2 = Shape(%1, start, end)
+//   %3 = Transpose(%1, perm)
+// into a special Op
+//   %2, %3 = ConcatShapeTranspose(inputs, axis, start, end, perm)
+// This fusion is an experimental work for performance
+
+// Helper function: is the ConcatOp matched to the fusion pattern?
+static bool isConcatFuseMatched(
+    Operation *op, ONNXShapeOp &shapeOp, ONNXTransposeOp &transposeOp) {
+  shapeOp = NULL;
+  transposeOp = NULL;
+  bool failed;
```

Comment:
@chentong319, this code caused the failures on Jenkins since `failed` is not intialized. It happens when building onnx-mlir with Release mode. Chaning this to `bool failed = false` will fix the failures.

Reply:
Thanks